src/PollController.js

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/FormController',
+],
+function (Okta, FormController) {
+  return FormController.extend({
+    className: 'poll',
+    Model: {
+      save: function () {
+        this.trigger('cancelRequest');
+        return this.appState.get('transaction').cancel()
+          .then(() => {
+            this.options.appState.trigger('navigate', '');
```

Comment:
Where does this take you  in the actual flow.  Just want to make sure you go back to the intended place.

Reply:
takes you back to the login page